src/pyFAI/gui/pilx/MainWindow.py

```diff
@@ -126,11 +128,12 @@ def initData(self,
 
         with h5py.File(self._file_name, "r") as h5file:
             nxprocess = h5file[self._nxprocess_path]
-            map = get_dataset(nxprocess, "result/intensity")[:, :, 0]
+            map = get_dataset(nxprocess, "result/intensity")[:,:, 0]
             pyFAI_config_as_str = get_dataset(
                 parent=nxprocess,
                 path=f"configuration/data"
             )[()]
+            self.worker_config = WorkerConfig.from_dict(json.loads(pyFAI_config_as_str), inplace=True)
```

Comment:
Is this used anywhere ? What is the purpose of storing the worker config ?

Reply:
The idea is to use the WorkerConfig at several places ... but it is not yet implemented (thus the WIP).
It should also host the shape of the input image and several other parameters.